iglu-server/0.4.0/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM snowplow-docker-registry.bintray.io/snowplow/base-alpine:0.2.0
```

Comment:
since we might want to deploy that in gcp too we might be better off with debian

Reply:
be aware that this means gosu and not su-exec, and not apk